packages/salesforcedx-sobjects-faux-generator/src/describe/sObjectDescribe.ts

```diff
@@ -29,24 +36,59 @@ export class SObjectDescribe {
    * @param type SObjectCategory
    * @returns string[] containing the sobject names
    */
-  public async describeGlobal(type: SObjectCategory): Promise<string[]> {
+  public async describeGlobal(
+    type: SObjectCategory,
+    source: SObjectRefreshSource
+  ): Promise<string[]> {
     const requestedDescriptions: string[] = [];
     const allDescriptions: DescribeGlobalResult = await this.connection.describeGlobal();
 
-    allDescriptions.sobjects.forEach((sobject: DescribeGlobalSObjectResult) => {
-      const isCustom = sobject.custom === true;
+    for (const sobject of allDescriptions.sobjects) {
```

Comment:
nit: can use `Array.prototype.filter` here

Reply:
I wasn't able to cause we also need to return a string (sobject.name) instead of the complete object

Reply:
`map` + `filter` or `reduce` could do it in one pass (Just a style preference. Fine with this too)

Reply:
I always forget about `reduce`, thanks for the suggestion I've change it to use that.